packages/lib-classifier/src/store/SubjectStore/shared/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './TextSubject.js'
```

Comment:
Should these files move to `store/SubjectStore/TextSubject`, for consistency with the other subject models?

Reply:
Yes I think that's a good idea, moved accordingly.